### DISCUSSION TITLE
Firefox ignoring break-after rules on printing

### DISCUSSION THREAD

**Discussion by GustavoOS** [tailwindlabs/tailwindcss#13805] [Help]
Firefox is ignoring break-after-page. The same behaviour is different on Chromium-based browsers.
Is it a bug in Firefox or in Tailwind?

**Comment by wongjn** [comment]
I would presume it would be some bug outside of Tailwind. This is because all that `break-after-page` does is:

```css
.break-after-page {
  break-after: page;
}
```
Nothing super Tailwind-specific in the rule.

**Comment by GustavoOS** [comment]
I found the answer for that one: it's a Firefox bug. https://bugzilla.mozilla.org/show_bug.cgi?id=1413121
